packages/fast-components-class-name-contracts-base/src/breadcrumb/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The class name contract for the breadcrumb component
+ */
+export interface BreadcrumbClassNameContract {
+    breadcrumb?: string;
+    breadcrumb_item?: string;
+    breadcrumb_item__current?: string;
+    breadcrumb_ol?: string;

```

Comment:
We tend to shy away from naming something after the element; is there something we can use for this instead of `ol`?